### ISSUE TITLE
Implement a StateHandler to manage CentralService instances

### ISSUE THREAD

**Issue by mdebarros** [mojaloop/project#229]
**Goal**:
As a developer I would like to __Implement a StateHandler to manage CentralService instances__ so that state can be managed across multiple instances of Central Services

**Tasks**:
- [ ] ...

**Acceptance Criteria**:
- [ ] Investigate best solution for StateHandler (e.g. kafka, reddis, etc)
- [ ] StateHandler to consume events that impact the state of the Central Service instance
- [ ] StateHandler to initialise state on startup of a Central Service instance
- [ ] StateHandler to consume from multiple topics (e.g. topic to store the Transfer Settlement Batch Id) or alternatively one topic that store a JSON object that contains all of the stateful values) -- to be confirmed/validated
- [ ] Update documentations (e.g. arch diagrams, etc): https://github.com/mdebarros/docs/tree/feature/%23147DocumentTheUpdatedDesignFlowForEnd-to-endPaymentProcess/CentralServices/ (link to be updated once the feature is merged into develop)

**Dependencies**:
- ...

**Comment by elnyry** [comment]
Not applicable right now, can be revived if this topic is taken up in future
